### DISCUSSION TITLE
1.1.0 new version is about to release

### DISCUSSION THREAD

**Discussion by VShawn** [1Remote/1Remote#818] [General]
Since the last official release, we have accumulated many updates: https://github.com/1Remote/1Remote/issues?q=is%3Aclosed+milestone%3ANEXT, so I think about to release a new version.

@itagagaki, are there any other issues that need to be resolved if I prepare to release a new version?

Unfortunately, progress on https://github.com/1Remote/1Remote/issues/559 has been quite slow. I haven't been able to achieve the same experience using RDP based on WinForm as we currently have - you can see details in the corresponding development branch.

**Comment by itagagaki** [comment]
For me, the only remaining work is the Options>Database glitch (which I'm sure will be fixed soon). Other than that, I have no other concerns. I think it is time for a good shipment :)

**Comment by VShawn** [comment]
okey i will release new version after we make a deal with https://github.com/1Remote/1Remote/pull/803#issuecomment-2558038420
